src/main/java/com/rhc/automation/Swagger2SpringBoot.java

```diff
@@ -0,0 +1,52 @@
+package com.rhc.automation;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.boot.CommandLineRunner;
+import org.springframework.boot.ExitCodeGenerator;
+import org.springframework.boot.SpringApplication;
+import org.springframework.boot.autoconfigure.SpringBootApplication;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.ComponentScan;
+
+import com.rhc.automation.service.CustomerService;
+import com.rhc.automation.service.OpenShiftService;
+import com.rhc.automation.service.UserService;
+
+import springfox.documentation.swagger2.annotations.EnableSwagger2;
+
+@SpringBootApplication
+@EnableSwagger2
+@ComponentScan(basePackages = "com.rhc.automation")
+public class Swagger2SpringBoot implements CommandLineRunner {
+    private static final Logger log = LoggerFactory.getLogger(Swagger2SpringBoot.class);

```

Comment:
I removed this logger since it was not being used. Also removed other unused code